Doc/library/sqlite3.rst

```diff
@@ -94,6 +94,12 @@ using :meth:`~Cursor.executemany`::
    ... ]
    >>> cur.executemany('INSERT INTO stocks VALUES(?, ?, ?, ?, ?)', data)
 
+Notice that we used ``?`` placeholders to bind *data* to the query.
+Always use placeholders instead of :ref:`string formatting<tut-formatting>`
```

Comment:
```suggestion
Always use placeholders instead of :ref:`string formatting <tut-formatting>`
```
Weird, `sphinx-lint` should have caught this :thinking:

Reply:
Yes, that valid, but I agree it looks weird. +1 for consistency!

Reply:
As I just mentioned in https://github.com/python/cpython/pull/95626#pullrequestreview-1062529930, there is at least another occurrence of this in the file.  You might want to check/fix them all in a separate PR.

Reply:
Thanks for the heads-up!